azure-pipelines.yml

```diff
@@ -18,6 +21,12 @@ jobs:
   - checkout: self
     fetchDepth: 64
 
+  - script: |
+      if [ ! -z "$(xcodeApp)" ]; then
```

Comment:
`! -z` is more simply written as `-n`.

It could even be a one liner:

```
[ -n "$(xcodeApp)" ] && sudo xcode-select --switch "$(xcodeApp)"
```

Reply:
Thanks for the tip. Since `xcode-select --switch` is quiet, I have added some informational output.